lightning/src/ln/channel.rs

```diff
@@ -4354,28 +4355,43 @@ mod tests {
 		PublicKey::from_secret_key(&secp_ctx, &SecretKey::from_slice(&hex::decode(hex).unwrap()[..]).unwrap())
 	}
 
-	#[test]
-	fn channel_reestablish_no_updates() {
-		let feeest = TestFeeEstimator{fee_est: 15000};
-		let logger = test_utils::TestLogger::new();
-		let secp_ctx = Secp256k1::new();
-		let mut seed = [0; 32];
-		let mut rng = thread_rng();
-		rng.fill_bytes(&mut seed);
-		let network = Network::Testnet;
-		let keys_provider = test_utils::TestKeysInterface::new(&seed, network);
+	struct ChanCfg {
+		fee_est: TestFeeEstimator,
+		logger: test_utils::TestLogger,
+		keys_mgr: test_utils::TestKeysInterface,
+	}
+
+	fn create_chan_cfgs(num_chans: usize) -> Vec<ChanCfg> {
+		let mut chan_cfgs = Vec::new();
+		for _ in 0..num_chans {
+			let fee_est = TestFeeEstimator{fee_est: 250 };
+			let logger = test_utils::TestLogger::new();
+			let mut seed = [0; 32];
+			let mut rng = thread_rng();
+			rng.fill_bytes(&mut seed);
+			let network = Network::Testnet;
+			let keys_mgr = test_utils::TestKeysInterface::new(&seed, network);
+			chan_cfgs.push(ChanCfg{ fee_est, logger, keys_mgr });
+		}
 
+		chan_cfgs
+	}
+
+	fn create_channels(cfgs: &Vec<ChanCfg>, amt: u64, push_amt_msat: u64) -> Vec<Channel<EnforcingChannelKeys>> {
```

Comment:
Consider naming `create_channel` and returning a tuple if this is suppose to give two views of the same channel (i.e. from different sides). Then you could possibly assign the return value as:

```rust
let (local_channel, remote_channel) = create_channel(...);
```

Or some more suitable names.

Reply:
I changed it to a tuple but I'm not sure what variable names would be more descriptive.